src/components/molecules/TodoBottomPanel.vue

```diff
@@ -0,0 +1,49 @@
+<script setup>
+import TaskCounter from "@/components/atoms/TaskCounter.vue";
+import FilterButton from "@/components/atoms/FilterButton.vue";
+import ClearCompletedButton from "@/components/atoms/ClearCompletedButton.vue";
+import { useStore } from "vuex";
+
+const store = useStore();
+
+function setFilter(filterName) {
+  store.commit("setFilter", filterName);
+}
+
+function deleteCompletedTasks() {
+  store.commit("removeAllCompleted");
+}
+</script>
+
+<template>
+  <div class="bottom-panel">
+    <TaskCounter />
+    <FilterButton
+      @click="setFilter('incomplete')"
```

Comment:
I think it's better to store these values ​​- incomplete, completed and all, somewhere as constants. You can create a new file constants.js and place it there